src/doc/rustc/src/platform-support/wasm32-unknown-emscripten.md

```diff
@@ -0,0 +1,143 @@
+# `wasm32-unknown-emscipten`
+
+**Tier: 2**
+
+The `wasm32-unknown-emscripten` target is a WebAssembly compilation target which
+uses the [Emscripten](https://emscripten.org/) compiler toolchain. Emscripten is
+a C/C++ toolchain used for running C/C++ code on the web. It provides POSIX `libc`
+and `libstd` implementations, access to the OpenGL and SDL APIs, and the ability
```

Comment:
It implements most of posix in a mostly compatible way, and also a lot of linux APIs. It might be worth saying here that the design goal is to allow C/C++ code (e.g., 3d games) written to target linux to be ported with as few changes as possible. Also, it uses musl libc.

Reply:
I've adapted the wording.